airflow/utils/log/secrets_masker.py

```diff
@@ -144,6 +144,11 @@ def _record_attrs_to_ignore(self) -> Iterable[str]:
         )
         return frozenset(record.__dict__).difference({'msg', 'args'})
 
+    def _redact_exception_with_context(self, exception):
+        exception.args = (self.redact(v) for v in exception.args)
+        if exception.__context__:
```

Comment:
Do we need a `hasattr`/`getattr` here, or do all exceptions have this field?

Reply:
Oh there's also a `__cause__` attr that we should probably redact too.

Reply:
Oh there's also a `__cause__` attr that we should probably redact too.

Reply:
Yeah. https://www.python.org/dev/peps/pep-3134/ . I never before saw the "from" syntax, but yeah... __cause__ should be there as well.